tools/subgraph-example/scripts/hts-transfer.ts

```diff
@@ -35,17 +35,24 @@ export async function transferHtsFT(receiver: string, hre: any) {
   const tokenId = TokenId.fromSolidityAddress(networks.default.local.ExampleHTSFT.address);
   const accountId = AccountId.fromString("0.0.1013");
 
-  let associateTx = await new TokenAssociateTransaction()
-            .setAccountId(accountId)
-            .setTokenIds([tokenId])
-            .freezeWithSigner(wallet)
-  await associateTx.sign(privateKey);
-  await (
+  try {
+    let associateTx = await new TokenAssociateTransaction()
+        .setAccountId(accountId)
+        .setTokenIds([tokenId])
+        .freezeWithSigner(wallet)
+    await associateTx.sign(privateKey);
     await (
-        await associateTx.signWithSigner(wallet)
-    ).executeWithSigner(wallet)
-  ).getReceiptWithSigner(wallet);
-
+        await (
+            await associateTx.signWithSigner(wallet)
+        ).executeWithSigner(wallet)
+    ).getReceiptWithSigner(wallet);
+  } catch (error) {
+    if(error instanceof ReceiptStatusError && error.status._code === 194) {
+      console.log("Token already associated with account")
+    } else {
+      console.log(error);
```

Comment:
Should this throw or is not throwing and not doing any extra logic preferred?

Reply:
Its almost the same.
1. Association fails for any other reason than `Already Associated` (code `194`) 
     a. but it was already associated, is preferable to continue, so the graph-example will work as expected.
     b. was not already associated, will fail on the next step when attempting to perform the transfer, probably with error, account is not associated.